scripts/publish.sh

```diff
@@ -1,20 +1,22 @@
 #!/bin/bash
 
 # When publishing the plugin
-#  1. Update the version by running `npm version`
+#  1. Update the version by running `npm version <major|minor|patch>`
 #  2. Run this script
 #  3. Commit the svn repo
 
 # echo "usage: ./scripts/publish.sh <svn dir>"
 
 VER=`./scripts/getversion.js`
 
+npm install
 npm run build
-sed -i "s/'build\/'/'assets\/'/g" wikipediapreview.php
 cp ./wikipediapreview.php $1/trunk/
 cp ./readme.txt $1/trunk/
-cp ./build/{index.js,init.js,style-index.css} $1/trunk/assets
-cp ./assets/{wikipedia-preview-link.css,wikipedia-preview.production.js} $1/trunk/assets
+mkdir $1/trunk/build
+cp ./build/{index.js,init.js,style-index.css} $1/trunk/build
+mkdir $1/trunk/libs
+cp ./libs/{wikipedia-preview-link.css,wikipedia-preview.production.js} $1/trunk/libs
```

Comment:
`libs` contains everything we need there, can copy entire folder so we don't manually add them.

Reply:
That's true but I prefer to be explicit with the publish script for now.